src/components/application_manager/src/commands/mobile/reset_global_properties_request.cc

```diff
@@ -131,13 +131,18 @@ void ResetGlobalPropertiesRequest::Run() {
       }
       msg_params = *vr_help;
     }
-    if (menu_name) {
-      msg_params[hmi_request::menu_title] = "";
+    if (menu_title) {
+      msg_params[hmi_request::menu_title] =
+          application_manager_.get_settings().menu_title();
       app->set_menu_title(msg_params[hmi_request::menu_title]);
```

Comment:
@dtrunov Clarifyed: it is expected by Ford that all values in .ini file shall be valid, and validation of .ini file is out of scope of the CRQ. So there should be no problems in reading or resetting of this parameter.